packages/flame/test/sprite_test.dart

```diff
@@ -0,0 +1,50 @@
+import 'dart:ui';
+
+import 'package:flame/components.dart';
+import 'package:flame/game.dart';
+import 'package:flame_test/flame_test.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_test/flutter_test.dart';
+
+import '_assets/images.dart';
+
+void main() {
+  group('Sprite', () {
+    FlameTester(() => FlameGame()).testGameWidget(
```

Comment:
```suggestion
    FlameTester(FlameGame.new).testGameWidget(
```

Reply:
This requires Dart 2.15+, and we're still on 2.14.

Reply:
Let's uuupgrade! 🤩

Reply:
I'm all for it, but has to be in a separate PR though